### DISCUSSION TITLE
Huge Security flag

### DISCUSSION THREAD

**Discussion by ForsakenTwilight** [vladmandic/sdnext#3366] [General]
My crypto was stolen just one day after I installed Vladmandic, not sure how it happened.

But when it was first installed I got an AV prompt, which I thought was a false positive
But Apparently I was wrong.
Never trusting any of you again
and this was already reported to the feds, I know nothing will come out of it, but hopefully enough people report it

**Comment by vladmandic** [comment]
sdnext does not include ANY binaries or compiled code and entire source code is here on GitHub where anyone can see it.

If you have any more information other than a wild claim, I'd be happy to investigate further.

**Comment by ForsakenTwilight** [comment]
i'll repost the same reply here as well.

I really don't know what to tell you.
All I can say is, I downloaded this and configured it on July 27, during which I got an AV prompt from Microsoft windows security, which I allowed on my computer back then, as this was supposed to be "safe" (now i know its not) and that it should be merely a false positive (granted i was using vlad diffusion before this without issues).
The next day, on july 28, all my crypto was gone (completely emptied from my wallet), my crypto was sitting in that wallet for years, and the only difference was installing vladmantic.

This is the gist of what happened.

**Reply by drax-xard** [reply]
What an elaborate way to advertise!

**Comment by vladmandic** [comment]
let's use the issue to continue, no need for reposting anymore.
